spec.html

```diff
@@ -3742,7 +3674,7 @@ <h1>The List and Record Specification Types</h1>
       <p>When an algorithm iterates over the elements of a List without specifying an order, the order used is the order of the elements in the List.</p>
       <p>For notational convenience within this specification, a literal syntax can be used to express a new List value. For example, &laquo; 1, 2 &raquo; defines a List value that has two elements each of which is initialized to a specific value. A new empty List can be expressed as &laquo; &raquo;.</p>
       <p>In this specification, the phrase "the <dfn id="list-concatenation">list-concatenation</dfn> of _A_, _B_, ..." (where each argument is a possibly empty List) denotes a new List value whose elements are the concatenation of the elements (in order) of each of the arguments (in order).</p>
-      <p>The <dfn>Record</dfn> type is used to describe data aggregations within the algorithms of this specification. A Record type value consists of one or more named fields. The value of each field is either an ECMAScript value or an abstract value represented by a name associated with the Record type. Field names are always enclosed in double brackets, for example [[Value]].</p>
+      <p>The <dfn>Record</dfn> type is used to describe data aggregations within the algorithms of this specification. A Record type value consists of one or more named fields. The value of each field is either an ECMAScript value or an abstract value represented by a name associated with the Record type. Field names are always enclosed in double brackets, for example [[Value]]. When the name of a field is identified by a named alias (<emu-xref href="#sec-algorithm-conventions"></emu-xref>), the alias name is wrapped in angle brackets inside the double brackets, for example [[&lt;_fieldName_&gt;]].</p>
```

Comment:
Is this a thing we actually do?

Reply:
It currently exists as an extension in [ECMA-402](https://tc39.es/ecma402/#conventions), and I'm pulling it upstream.